Add --context-dir option to podman play kube (#12913)
containers/podman: docs/source/markdown/podman-play-kube.1.md

#### **--build**

Build images even if they are found in the local storage.  (This option is not available with the remote Podman client)

#### **--cert-dir**=*path*

Use certificates at *path* (\*.crt, \*.cert, \*.key) to connect to the registry. (Default: /etc/containers/certs.d)
Please refer to containers-certs.d(5) for details. (This option is not available with the remote Podman client)

> #### **--context-dir**=*path*

Reviewer: Are these supposed to be listed alphabetically?

Reviewer: This should be after _--configmap_, then.

Author: Moved

> Use context-dir to find build context directory for each image. Requires --build option be true.  (This option is not available with the remote Podman client)

Reviewer 2: ```suggestion
Use context-dir to find the build context directory for each image. Requires --build option be true. (This option is not available with the remote Podman client)
```

> Use context-dir to find the build context directory for each image. Requires --build option be true. (This option is not available with the remote Podman client)

Reviewer: ```suggestion
Use *path* as the build context directory for each image. Requires --build option be true. (This option is not available with the remote Podman client)
```
